admin-dev/themes/new-theme/js/app/widgets/ps-table/ps-sort.vue

```diff
@@ -23,30 +23,44 @@
  * International Registered Trademark & Property of PrestaShop SA
  *-->
 <template>
-  <button class="ps-sort" @click.prevent="sortFilter">
-    <i class="material-icons">swap_vert</i>
-  </button>
+  <div class="ps-sortable-column" data-sort-col-name="id_product" :data-sort-is-current="isCurrent" :data-sort-direction="sortDirection" @click="sortToggle">
+    <span role="columnheader"><slot /></span>
+    <span role="button" class="ps-sort" aria-label="Tri"></span>
+  </div>
 </template>
 
 <script>
   export default {
-    props: ['order'],
+    props: {
+      // column name
+      order: String,
+      // indicates the currently sorted column in the table
+      currentSort: {
```

Comment:
If not required, use same syntax as above. `currentSort: String`

Reply:
👍